### DISCUSSION TITLE
GPU cooling optimisation

### DISCUSSION THREAD

**Discussion by Demcrew** [Rem0o/FanControl.Releases#1996] [Q&A] [answered]
Hi Rémi, 

First of all thanks a million for this golden pieace of software. I really hope you'll be able to keep maintaining it. All the good lads isolated in their homes with their PC, appreciate the relief provided by your software. Especially in this ever warmer environment.

Nowadays, we can consider Graphic cards from 2 differents angles, the GPU itself and the Memory. 
My issue is the following: 
When playing games where the Memory is heavily involved, the temp of the Memory can hit 90° C, while the GPU is only at 56-58°.
So for now I've setup the curve accordingly to mitigate the heat of the Memory.

However, it's a bit annoying, as soon as the GPU hit this temp, well... Fans are doing their job, no matter what the Memory temp is...
And some scenarios definitely don't deserve that spike of the fans...

It seems that I couldn't manage to get a sensor for this particular aspect.

How can I optimise this ?

**Comment by tall1on** [comment]
Which GPU do you have? 90°C sounds like RTX3090 because of the backplate memory?

**Reply by Demcrew** [reply]
You guessed close to it, I have an MSI RTX 3080 Gaming Z Trio 10G LHR

**Reply by Demcrew** [reply]
Here is my Fan Control Setup
![image](https://github.com/Rem0o/FanControl.Releases/assets/35411966/a0b340b6-6827-40cd-9918-3a94247c86c7)

And here is a logic view of the setup
![image](https://github.com/Rem0o/FanControl.Releases/assets/35411966/290d1a2b-cb91-41b5-94d0-f4f2a4ff4c08)

**Reply by tall1on** [reply, marked as answer]
As far as i know, only the RTX3090 has backplate memory. My old RTX 3090 vision had 90-105°C under heavy load. Generally i think up to 110°C is safe for the chips on RTX3000, but i can understand that you want the temps lower.

I guess the issue here also is, that FanControl does not support your memory temp sensor, but as the Readme says, its impossible to support every piece of hardware, because you need to own all of it to test it.

One idea i have is the following: You could attach a thermal probe to the backplate of your card (at the position of the vram, use for example kapton tape) and connect it to a motherboard temp header if you got one. Motherboard sensors are usally easier to support because they use standardized control chips and maybe you could build a curve with the data then.

**Reply by Demcrew** [reply]
Agreeing on all the 3 points mentioned. Indeed, I'm still on the safe side around 90°. 
Also I understand Fan Control cannot deal with the whole variety of hardware, I'm actually very impressed by its current versatility on this aspect.
However, I was able to figure out the VRAM temp via the clunky Asus Crate  software... (I've installed and removed a bunch of tools just to refine the information that I have from the system), so I was wondering if Crate could do it, then why not Fan Control ;)

I'll try go for the latter and get a sensor settled as you suggested.

I must say thank you for taking the time to respond! :)

**Reply by Demcrew** [reply]
Not marking it as answer, in the hope Rémi would have a look at this thread.

**Reply by Rem0o** [reply]
I did see this thread, but as the README states, I can't dev on hardware I don't have.  

However, have you tried to use the LHM built-in GPU sensors instead of NVApiWrapper? Try to deactivate NvAPIWrapper, it will mess your config, but check if you got more temp sensors.

**Reply by Demcrew** [reply]
Okay I'll give it a try and will let you know shortly :)

**Reply by Demcrew** [reply]
Hi Remi, 

Thanks for taking the time to respond :) appreciated! 
Obviously, you can't dev for all hardware, I totally got it, no worries at all. 

I've seen the latest release, thanks for the update!

I've tried what you suggested and indeed the config has been a bit screwed up, thanks for the profile backup :)
I didn't get the temp sensor that I needed unfortunately. 

I just found a thermal probe I had, hiding in a box, but unfortunately the motherboard don't have a slot to plug it into. 

I'll try fine tune the setup the best I can, it is already far better than previsouly, because of Fan Control.

Kudos to you really, thanks for all your hard work!
